2016/08/09 23:30:40 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [1049] Unknown database 'alelo'' in C:\wamp64\www\alelo\framework\db\CDbConnection.php:399
Stack trace:
#0 C:\wamp64\www\alelo\framework\db\CDbConnection.php(347): CDbConnection->open()
#1 C:\wamp64\www\alelo\framework\db\CDbConnection.php(325): CDbConnection->setActive(true)
#2 C:\wamp64\www\alelo\framework\base\CModule.php(394): CDbConnection->init()
#3 C:\wamp64\www\alelo\framework\base\CModule.php(103): CModule->getComponent('db')
#4 C:\wamp64\www\alelo\framework\gii\generators\model\ModelCode.php(60): CModule->__get('db')
#5 C:\wamp64\www\alelo\framework\web\CFormModel.php(39): ModelCode->init()
#6 C:\wamp64\www\alelo\framework\gii\CCodeGenerator.php(153): CFormModel->__construct()
#7 C:\wamp64\www\alelo\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 C:\wamp64\www\alelo\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 C:\wamp64\www\alelo\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\wamp64\www\alelo\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\wamp64\www\alelo\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\wamp64\www\alelo\framework\web\CWebApplication.php(282): CController->run('index')
#13 C:\wamp64\www\alelo\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 C:\wamp64\www\alelo\framework\base\CApplication.php(185): CWebApplication->processRequest()
#15 C:\wamp64\www\alelo\alelo\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/alelo/alelo/index.php/gii/model/index
HTTP_REFERER=http://localhost/alelo/alelo/index.php/gii/default/index
---
2016/08/16 23:26:50 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in C:\wamp64\www\alelo\framework\db\CDbConnection.php:399
Stack trace:
#0 C:\wamp64\www\alelo\framework\db\CDbConnection.php(347): CDbConnection->open()
#1 C:\wamp64\www\alelo\framework\db\CDbConnection.php(325): CDbConnection->setActive(true)
#2 C:\wamp64\www\alelo\framework\base\CModule.php(394): CDbConnection->init()
#3 C:\wamp64\www\alelo\framework\base\CModule.php(103): CModule->getComponent('db')
#4 C:\wamp64\www\alelo\framework\gii\generators\model\ModelCode.php(60): CModule->__get('db')
#5 C:\wamp64\www\alelo\framework\web\CFormModel.php(39): ModelCode->init()
#6 C:\wamp64\www\alelo\framework\gii\CCodeGenerator.php(153): CFormModel->__construct()
#7 C:\wamp64\www\alelo\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 C:\wamp64\www\alelo\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 C:\wamp64\www\alelo\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\wamp64\www\alelo\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\wamp64\www\alelo\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\wamp64\www\alelo\framework\web\CWebApplication.php(282): CController->run('index')
#13 C:\wamp64\www\alelo\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 C:\wamp64\www\alelo\framework\base\CApplication.php(185): CWebApplication->processRequest()
#15 C:\wamp64\www\alelo\Rafael-Oliveira\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/alelo/Rafael-Oliveira/index.php/gii/model/index
HTTP_REFERER=http://localhost/alelo/Rafael-Oliveira/index.php/gii/default/index
---
2016/08/16 23:30:02 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in C:\wamp64\www\alelo\framework\db\CDbConnection.php:399
Stack trace:
#0 C:\wamp64\www\alelo\framework\db\CDbConnection.php(347): CDbConnection->open()
#1 C:\wamp64\www\alelo\framework\db\CDbConnection.php(325): CDbConnection->setActive(true)
#2 C:\wamp64\www\alelo\framework\base\CModule.php(394): CDbConnection->init()
#3 C:\wamp64\www\alelo\framework\base\CModule.php(103): CModule->getComponent('db')
#4 C:\wamp64\www\alelo\framework\gii\generators\model\ModelCode.php(60): CModule->__get('db')
#5 C:\wamp64\www\alelo\framework\web\CFormModel.php(39): ModelCode->init()
#6 C:\wamp64\www\alelo\framework\gii\CCodeGenerator.php(153): CFormModel->__construct()
#7 C:\wamp64\www\alelo\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 C:\wamp64\www\alelo\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 C:\wamp64\www\alelo\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\wamp64\www\alelo\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\wamp64\www\alelo\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\wamp64\www\alelo\framework\web\CWebApplication.php(282): CController->run('index')
#13 C:\wamp64\www\alelo\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 C:\wamp64\www\alelo\framework\base\CApplication.php(185): CWebApplication->processRequest()
#15 C:\wamp64\www\alelo\Rafael-Oliveira\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/alelo/Rafael-Oliveira/index.php/gii/model/index
---
2016/08/16 23:35:40 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[22007]: Invalid datetime format: 1292 Incorrect date value: '' for column 'deleted_at' at row 1. The SQL statement executed was: INSERT INTO `mission` (`name`, `description`, `goup_id`, `deleted_at`) VALUES (:yp0, :yp1, :yp2, :yp3).
in C:\wamp64\www\alelo\Rafael-Oliveira\protected\controllers\MissionController.php (73)
in C:\wamp64\www\alelo\Rafael-Oliveira\index.php (13)
2016/08/16 23:35:40 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[22007]: Invalid datetime format: 1292 Incorrect date value: '' for column 'deleted_at' at row 1. The SQL statement executed was: INSERT INTO `mission` (`name`, `description`, `goup_id`, `deleted_at`) VALUES (:yp0, :yp1, :yp2, :yp3)' in C:\wamp64\www\alelo\framework\db\CDbCommand.php:358
Stack trace:
#0 C:\wamp64\www\alelo\framework\db\ar\CActiveRecord.php(1080): CDbCommand->execute()
#1 C:\wamp64\www\alelo\framework\db\ar\CActiveRecord.php(809): CActiveRecord->insert(NULL)
#2 C:\wamp64\www\alelo\Rafael-Oliveira\protected\controllers\MissionController.php(73): CActiveRecord->save()
#3 C:\wamp64\www\alelo\framework\web\actions\CInlineAction.php(49): MissionController->actionCreate()
#4 C:\wamp64\www\alelo\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\wamp64\www\alelo\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 C:\wamp64\www\alelo\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\wamp64\www\alelo\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\wamp64\www\alelo\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\wamp64\www\alelo\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\wamp64\www\alelo\framework\web\CController.php(291): CFilterChain->run()
#11 C:\wamp64\www\alelo\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\wamp64\www\alelo\framework\web\CWebApplication.php(282): CController->run('create')
#13 C:\wamp64\www\alelo\framework\web\CWebApplication.php(141): CWebApplication->runController('mission/create')
#14 C:\wamp64\www\alelo\framework\base\CApplication.php(185): CWebApplication->processRequest()
#15 C:\wamp64\www\alelo\Rafael-Oliveira\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/alelo/Rafael-Oliveira/index.php/mission/create
HTTP_REFERER=http://localhost/alelo/Rafael-Oliveira/index.php/mission/create
---
2016/08/17 00:34:10 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "user".' in C:\wamp64\www\alelo\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\wamp64\www\alelo\framework\web\CWebApplication.php(141): CWebApplication->runController('user')
#1 C:\wamp64\www\alelo\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\wamp64\www\alelo\Rafael-Oliveira\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/alelo/Rafael-Oliveira/index.php/user
---
2016/08/17 00:34:16 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "user/user".' in C:\wamp64\www\alelo\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\wamp64\www\alelo\framework\web\CWebApplication.php(141): CWebApplication->runController('user/user')
#1 C:\wamp64\www\alelo\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\wamp64\www\alelo\Rafael-Oliveira\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/alelo/Rafael-Oliveira/index.php/user/user
---
2016/08/17 00:35:25 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[22007]: Invalid datetime format: 1292 Incorrect date value: '10/10/2016' for column 'created_at' at row 1. The SQL statement executed was: INSERT INTO `user` (`username`, `password`, `created_at`, `deleted_at`, `last_login`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4).
in C:\wamp64\www\alelo\Rafael-Oliveira\protected\controllers\UserController.php (73)
in C:\wamp64\www\alelo\Rafael-Oliveira\index.php (13)
2016/08/17 00:35:25 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[22007]: Invalid datetime format: 1292 Incorrect date value: '10/10/2016' for column 'created_at' at row 1. The SQL statement executed was: INSERT INTO `user` (`username`, `password`, `created_at`, `deleted_at`, `last_login`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4)' in C:\wamp64\www\alelo\framework\db\CDbCommand.php:358
Stack trace:
#0 C:\wamp64\www\alelo\framework\db\ar\CActiveRecord.php(1080): CDbCommand->execute()
#1 C:\wamp64\www\alelo\framework\db\ar\CActiveRecord.php(809): CActiveRecord->insert(NULL)
#2 C:\wamp64\www\alelo\Rafael-Oliveira\protected\controllers\UserController.php(73): CActiveRecord->save()
#3 C:\wamp64\www\alelo\framework\web\actions\CInlineAction.php(49): UserController->actionCreate()
#4 C:\wamp64\www\alelo\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\wamp64\www\alelo\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 C:\wamp64\www\alelo\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\wamp64\www\alelo\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\wamp64\www\alelo\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\wamp64\www\alelo\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\wamp64\www\alelo\framework\web\CController.php(291): CFilterChain->run()
#11 C:\wamp64\www\alelo\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\wamp64\www\alelo\framework\web\CWebApplication.php(282): CController->run('create')
#13 C:\wamp64\www\alelo\framework\web\CWebApplication.php(141): CWebApplication->runController('user/create')
#14 C:\wamp64\www\alelo\framework\base\CApplication.php(185): CWebApplication->processRequest()
#15 C:\wamp64\www\alelo\Rafael-Oliveira\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/alelo/Rafael-Oliveira/index.php/user/create
HTTP_REFERER=http://localhost/alelo/Rafael-Oliveira/index.php/user/create
---
